composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require": {
-        "tightenco/collect": "^5.3"
+        "tightenco/collect": "^5.3",
+		"s1lentium/iptools": "^1.1"
```

Comment:
The latest commit in this library has been tagged at 1.1.1 so using * previously here isn't necessary/a great practice